db/schema.rb

```diff
@@ -165,7 +165,7 @@
     t.date     "date_of_injury"
     t.string   "current_occupation"
     t.integer  "residence_type"
-    t.boolean  "has_caregiver"
+    t.integer  "has_caregiver"
```

Comment:
erp? 

It looks like you did this by hand....in general one should only ever touch schema.rb via migrations...